docs/example.yaml

```diff
@@ -16,6 +16,9 @@ spec:
       kind: StatefulSet
   version: "4.9.4"
   description: "WordPress is open source software you can use to create a beautiful website, blog, or app."
+  icons:
```

Comment:
I would include multiple icons and the size field in the example

Reply:
@ant31 FYI, I plan on opening a second PR that introduces the purpose field. Doing it as a separate PR so we can discuss/debate that single point. purpose is part of the web app manifest... https://www.w3.org/TR/appmanifest/#dom-imageresource-purpose

Reply:
Adding with real examples.